pkg/local_instance.go

```diff
@@ -219,6 +223,19 @@ func (li LocalInstance) Create() error {
 	return nil
 }
 
+func (li LocalInstance) Import() error {
+	log.Infof("%s > importing", li.instance.ID())
+
+	if err := pathx.Ensure(li.Dir()); err != nil {
+		return fmt.Errorf("%s > cannot create its dir: %w", li.instance.ID(), err)
+	}
+	if err := li.createLock().Lock(); err != nil {
```

Comment:
ok, but what if instance is already running... ;) 

you did some assumptions; so you should at least inform the user that on next start command the instance will be restarted because there is no up/start lock created

Reply:
I check if unpack dir exists and contains crx-quickstart

Reply:
It is is needed because you don't know with what runmode imported instance is currently running

Reply:
right. Added restart for running imported instances